Makefile

```diff
@@ -15,3 +15,14 @@ ifeq ($(VERSION),)
 endif
 	make VERSION=$(VERSION) bump && git commit -am "v$(VERSION)" && git tag v$(VERSION) \
 	&& git push origin && git push --tags && npm publish
+
+upgrade_vendor:
+ifeq ($(VERSION),)
+	@$(error VERSION is not defined. Run with `make VERSION=number upgrade_vendor`)
+endif
+	@sed -i '' "s/bugsnag-android:.*\'/bugsnag-android:$(VERSION)\'/" android/build.gradle
```

Comment:
I'd define a new variable for this. It wasn't clear that by running `make VERSION=foo bump upgrade_vendor` would set the android version to `foo`.